pkg/backup/client.go

```diff
@@ -405,13 +405,14 @@ func (bc *Client) BackupRanges(
 	allFiles := make([]*kvproto.File, 0, len(ranges))
 	go func() {
 		init := time.Now()
-		start, cur := init, init
+		lastBackupStart, currentBackupStart := init, init
 		for files := range filesCh {
-			cur, start = start, time.Now()
+			lastBackupStart, currentBackupStart = currentBackupStart, time.Now()
 			allFiles = append(allFiles, files...)
-			summary.CollectSuccessUnit("backup ranges", 1, cur.Sub(start))
+			summary.CollectSuccessUnit("backup ranges", 1, currentBackupStart.Sub(lastBackupStart))
 		}
-		log.Info("Backup Ranges", zap.Duration("take", cur.Sub(init)))
+		_ = lastBackupStart
```

Comment:
🤔️

Reply:
OK. Let me replace it with `// nolint`...